cppcache/src/TcrHADistributionManager.cpp

```diff
@@ -36,7 +38,7 @@ TcrHADistributionManager::TcrHADistributionManager(
     ThinClientRegion* theRegion, TcrConnectionManager& connManager)
     : ThinClientDistributionManager(connManager, theRegion),
       m_theTcrConnManager(connManager) {
-  GF_R_ASSERT(theRegion != nullptr);
+  assert(theRegion != nullptr);
```

Comment:
Why delete everywhere else but here? There were a few others that you converted to `assert`, why?

Reply:
GF_D_ASSERT, as defined, was always *off* by default, so I removed it.  GF_R_ASSERT was always *on*, so I replaced those with asserts to minimize impact on actual behavior of the code

Reply:
Make sense. As much as I don't like `asserts` in productions code, that is outside the scope of this issue. Good call!

Reply:
I kinda expect we can safely get rid of the 3 instances of it altogether, but yeah we should have a quick chat about that later